exampleCustomVideoCapturer/src/main/java/com/twilio/video/examples/customcapturer/ViewCapturer.java

```diff
@@ -61,7 +61,8 @@ public void run() {
 
                 // Create video frame
                 VideoDimensions dimensions = new VideoDimensions(view.getWidth(), view.getHeight());
-                VideoFrame videoFrame = new VideoFrame(array, dimensions, 0, captureTimeNs);
+                VideoFrame videoFrame = new VideoFrame(array,
+                        dimensions, VideoFrame.RotationAngle.ROTATION_0, captureTimeNs);
```

Comment:
Yes, according to the docs. https://media.twiliocdn.com/sdk/android/video/releases/1.0.0-beta16/docs/com/twilio/video/VideoFrame.html